src/actor/ActorTemplate.js

```diff
@@ -205,34 +255,42 @@ exports = function(inherits) {
     }
 
     /**
-      * Set {@link Actor#vx} and {@link Actor#vy} to aim for the specified point, with the specified speed.
-      * @func Actor#headToward
-      * @arg {number} x
-      * @arg {number} y
-      * @arg {number} speed
-      */
+     * Set vx and vy to aim for the specified point, with the specified speed.
+     * @method ActorTemplate#headToward
```

Comment:
Also thought about a `moveTo` fn. Gets kind of messy though - it would either need to use animate and then not have valid `vx` `vy` values, or it would need a callback to stop the `vx` `vy` when the destination was reached. The callback would also have to force the `x` `y` values to match those passed in since velocity is not exact.